web/src/assets/locales/de.json

```diff
@@ -0,0 +1,217 @@
+    "weeks_short": "w",
+    "days_short": "t",
+    "hours_short": "h",
+    "min_short": "min",
+    "sec_short": "sek",
+    "not_started": "noch nicht gestartet"
+  },
+
+  "repo": {
+    "activity": "Aktivitäten",
+    "branches": "Branches",
+    "add": "Repository hinzufügen",
+    "user_none": "Diese Organisation / dieser Benutzer hat noch keine Repositories.",
+    "not_allowed": "Zugriff auf dieses Repository nicht erlaubt",
+
+    "enable": {
+      "reload": "Repositories neu laden",
+      "enable": "Aktivieren",
+      "enabled": "Bereits Aktiviert",
+      "success": "Repository aktiviert",
+      "list_reloaded": "Repository-Liste neu geladen"
+    },
+
+    "settings": {
+      "settings": "Einstellungen",
+      "not_allowed": "Zugriff auf die Einstellungen dieses Repositorys nicht erlaubt",
+
+      "general": {
+        "general": "Allgemein",
+        "project": "Projekt-Einstellungen",
+        "save": "Einstellungen speichern",
+        "success": "Repository-Einstellungen aktualisiert",
+
+        "pipeline_path": {
+          "path": "Pipeline-Pfad",
+          "default": "Standardmäßig: .woodpecker/*.yml -> .woodpecker.yml -> .drone.yml",
+          "desc": "Pfad zu deiner Pipeline-Konfiguration (z. B. <span class=\"bg-gray-300 dark:bg-dark-700 rounded-md px-1\">mein/pfad/</span>). Ordner sollten mit einem <span class=\"bg-gray-300 dark:bg-dark-700 rounded-md px-1\">/</span> enden."
+        },
+
+        "allow_pr": {
+          "allow": "Pull Requests zulassen",
+          "desc": "Pipelines können für Pull-Requests genutzt werden."
+        },
+
+        "protected": {
+          "protected": "Geschützt",
+          "desc": "Jede Pipeline muss genehmigt werden, bevor sie ausgeführt wird."
+        },
+        "trusted": {
+          "trusted": "Vertrauenswürdig",
+          "desc": "Die zugrundeliegenden Pipeline-Container erhalten Zugriff auf ausgeweitete Funktionen wie z. B. das Einhängen von Volumes."
+        },
+        "visibility": {
+          "visibility": "Sichtbarkeit des Projekts",
+
+          "public": {
+            "public": "Öffentlich",
+            "desc": "Jeder Benutzer kann Ihr Projekt sehen, ohne eingeloggt zu sein."
+          },
+          "private": {
+            "private": "Privat",
+            "desc": "Nur Sie und andere Besitzer des Repositorys können dieses Projekt sehen."
+          },
+          "internal": {
+            "internal": "Intern",
+            "desc": "Nur authentifizierte Benutzer der Woodpecker-Instanz können dieses Projekt sehen."
+          }
+        },
+
+        "timeout": {
+          "timeout": "Zeitlimit",
+          "minutes": "Minuten"
+        },
+
+        "cancel_prev": {
+          "cancel": "Breche ältere Pipelines ab",
+          "desc": "Aktiviere diese Option, um laufende Pipelines desselben Ereignisses und Kontexts abzubrechen, bevor die neue Pipeline startet."
+        }
+      },
+
+      "secrets": {
+        "secrets": "Geheimnisse",
```

Comment:
I am not sure if we should call it `Geheimnisse`. Secrets is a pretty well known term in this topic IMO and calling secrets Geheimnisse feels to me like calling pipeline `Rohrleitung` 😉

Reply:
oh das hätte was :laughing: